scripts/ci/npm/publish-libs.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+ROOT_DIR=${DIR}/../../..
+DIST_DIR=${ROOT_DIR}/dist/@alfresco
+PROJECTS_DIR=${ROOT_DIR}/projects
+
+for var in "$@"
+do
+    case "$var" in
+      --dryrun) RUN_DRY=true
+      echo -e "Run with --dryrun, therfore script won't publish packages\n" ;;
+    esac
+done
+
+VERSION_IN_PACKAGE_JSON=`node -p "require('$ROOT_DIR/package.json')".version;`;
+
+if [ "$TRAVIS_BRANCH" = "master" ] ; then
+  NEW_LIBRATY_VARSION=VERSION_IN_PACKAGE_JSON
+else
+  NEW_LIBRATY_VARSION="${VERSION_IN_PACKAGE_JSON}-${TRAVIS_BUILD_NUMBER}"
```

Comment:
typo in the words "Library" and "Version" (NEW_LIBRATY_VARSION)

Reply:
fixed